packages/vm/src/testing/instance.rs

```diff
@@ -25,6 +25,20 @@ pub fn mock_instance(
     )
 }
 
+pub fn mock_instance_with_failing_api(
+    wasm: &[u8],
+    contract_balance: &[Coin],
+) -> Instance<MockStorage, MockApi, MockQuerier> {
+    mock_instance_with_options(
+        wasm,
+        MockInstanceOptions {
+            contract_balance: Some(contract_balance),
+            api_failure: true,
```

Comment:
I followed this little logic flow down below.
Nice and relatively unintrusive update :+1:

Reply:
thanks! :)